### PR TITLE
preset JS_QUERY_POOL_SIZE to reserve JS query pool

### PR DESCRIPTION
**Pull request by Will-ShaoHua**
1. set JS_QUERY_POOL_SIZE to avoid
```
2024-04-26 01:04:40,066 [  10213]   WARN - #c.i.u.j.JBCefJSQuery - Set the property JBCefClient.Properties.JS_QUERY_POOL_SIZE to use JBCefJSQuery after the browser has been created
java.lang.IllegalStateException
	at com.intellij.ui.jcef.JBCefJSQuery.create(JBCefJSQuery.java:66)
	at software.aws.toolkits.jetbrains.core.explorer.webview.ToolkitWebviewBrowser.<init>(ToolkitLoginWebview.kt:118)
	at software.aws.toolkits.jetbrains.core.explorer.webview.ToolkitWebviewPanel.<init>(ToolkitLoginWebview.kt:89)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
	at com.intellij.platform.instanceContainer.instantiation.InstantiateKt$instantiate$4.invoke(instantiate.kt:74)
	at com.intellij.platform.instanceContainer.instantiation.InstantiateKt$instantiate$4.invoke(instantiate.kt:72)
	at com.intellij.platform.instanceContainer.instantiation.InstantiateKt.instantiate(instantiate.kt:278)
	at com.intellij.platform.instanceContainer.instantiation.InstantiateKt.instantiate(instantiate.kt:72)
	at com.intellij.serviceContainer.InstantiateKt.instantiateWithContainer(instantiate.kt:19)
	at com.intellij.serviceContainer.ServiceInstanceInitializer.createInstance$suspendImpl(ServiceInstanceIniti
```

2. revert part of dedupe done previously which may cause unexpected parent-children class referencing to each other

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Checklist
- [ ] My code follows the code style of this project
- [ ] I have added tests to cover my changes
- [ ] A short description of the change has been added to the **[CHANGELOG](https://github.com/aws/aws-toolkit-jetbrains/blob/master/CONTRIBUTING.md#contributing-via-pull-requests)** if the change is customer-facing in the IDE.
- [ ] I have added metrics for my changes (if required)
 
## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/util/JcefBrowserUtil.kt [hunk 1]
```diff
@@ -8,9 +8,13 @@ import com.intellij.openapi.util.Disposer
 import com.intellij.ui.jcef.JBCefApp
 import com.intellij.ui.jcef.JBCefBrowserBase
 import com.intellij.ui.jcef.JBCefBrowserBuilder
+import com.intellij.ui.jcef.JBCefClient
 
 fun createBrowser(parent: Disposable): JBCefBrowserBase {
-    val client = JBCefApp.getInstance().createClient()
+    val client = JBCefApp.getInstance().createClient().apply {
```
Comment (Will-ShaoHua) [inline plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/util/JcefBrowserUtil.kt:14]: think we should re-use these clients

Comment (Will-ShaoHua) [reply plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/util/JcefBrowserUtil.kt:14]: maybe consider it as a followup

File: plugins/core/jetbrains-community/src/software/aws/toolkits/jetbrains/core/webview/LoginBrowser.kt [hunk 2]
```diff
@@ -58,25 +49,6 @@ abstract class LoginBrowser(
...
-
-                if (authorization != null) {
-                    executeJS("window.ideClient.updateAuthorization(\"${userCodeFromAuthorization(authorization)}\")")
-                    currentAuthorization = authorization
-                    return@launch
-                }
-            }
-        }
-    }
-
```
Comment (Will-ShaoHua) [inline plugins/core/jetbrains-community/src/software/aws/toolkits/jetbrains/core/webview/LoginBrowser.kt:79]: cleanup

**Review by rli — APPROVED** [review]

**Pull request merged**